client/src/legacy/TabSet.js

```diff
@@ -46,22 +56,35 @@ $.entwine('ss', function($){
       });
     },
 
+    /**
+     * @func triggerLazyLoad
+     * @desc Find all the lazy loadable gridfield and nodes in the panel and trigger their reload.
+     * @param {Object} panel
+     */
+    triggerLazyLoad: function(panel) {
+      panel.find('.grid-field--lazy-loadable, .lazy-loadable').each((i, el) => {
```

Comment:
The `.lazy-loadable` class should be documented in the lazy loading part of the [relevant documentation](https://github.com/silverstripe/silverstripe-framework/blob/4/docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/02_CMS_Architecture.md#tabs)